### ISSUE TITLE
[CI] Compilation issues on 7.12 periodic

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch-hadoop#1669]
This CI build seems to be failing for a while due to compile issues:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch-hadoop+7.12+periodic
Latest today: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch-hadoop+7.12+periodic/96/console

```
10:02:18 * What went wrong:
10:02:18 Execution failed for task ':elasticsearch-hadoop-hive:compileItestJava'.
10:02:18 > Could not resolve all files for configuration ':elasticsearch-hadoop-hive:itestCompileClasspath'.
10:02:18 Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
10:02:18 Use '--warning-mode all' to show the individual deprecation warnings.
10:02:18    > Could not resolve org.pentaho:pentaho-aggdesigner-algorithm:5.1.5-jhyde.
10:02:18      Required by:
10:02:18 See https://docs.gradle.org/6.8/userguide/command_line_interface.html#sec:command_line_warnings
10:02:18          project :elasticsearch-hadoop-hive > org.apache.hive:hive-exec:1.2.1 > org.apache.calcite:calcite-core:1.2.0-incubating
10:02:18       > Could not resolve org.pentaho:pentaho-aggdesigner-algorithm:5.1.5-jhyde.
10:02:18 41 actionable tasks: 41 executed
10:02:18          > Could not get resource 'https://repo.spring.io/plugins-release/org/pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde/pentaho-aggdesigner-algorithm-5.1.5-jhyde.pom'.
10:02:18             > Could not GET 'https://repo.spring.io/plugins-release/org/pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde/pentaho-aggdesigner-algorithm-5.1.5-jhyde.pom'. Received status code 401 from server: Unauthorized
```

This doesn't seem to be a temporary issue since it seems to also have failed in the last few days as well.

**Comment by cbuescher** [comment]
Also seems to be a problem on the 6.8 job https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch-hadoop+6.8+periodic/109/console

**Comment by williamrandolph** [comment]
Digging around on Slack, it looks like @breskeby fixed this issue on 6.8 in this commit: https://github.com/elastic/elasticsearch-hadoop/commit/35b09469bde6ac18370131075a5cb03bb63a1cdf

Now it looks like we have different failures. Rene, can you confirm that you fixed this particular issue?

**Comment by breskeby** [comment]
checking

**Comment by breskeby** [comment]
Ah I see what happened. I fixed this for 6.8 assuming that the repo wasn't used anymore in future branches but turns out its declaration has just moved into a plugin. Working on a fix for master that we can then cherry pick down to 7.x, 7.13, 7.12

**Comment by breskeby** [comment]
I've pushed a fix and back ported it to 7.x, 7.13, 7.12
